### PR TITLE
Tweak the README a bit and add a section on evolution process.

### PR DESCRIPTION
**Pull request by munificent**
What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,13 +74,21 @@ pre-defined lint sets provided by the lints package:
...
+    change here means adding or removing a lint from one or both sets. If you
+    want to discuss implementing an entirely new lint, the place to suggest that
+    is at the [linter repo].) Feedback is welcome from any Dart user.
 
-Feedback is welcome! If you want to propose an existing lint for inclusion here,
-check to see if there is already an open [issue] and chime in.  If there is not,
-please start a conversation with a new issue. If you have questions about existing
-lints or have an idea for a new one, please file an issue on the [linter repo].
+2.  Periodically, a group of Dart and Flutter team members meet to review the
+    suggests and decide what to adopt.
```
Comment (devoncarew) [inline README.md:89]: DBC: `suggests` ==> `suggestions`

File: README.md [hunk 2]
```diff
@@ -39,26 +36,26 @@ For new apps created with `dart create`, the lints are enabled by default.
...
-include: package:lints/core.yaml
-```
+    ```yaml
+    include: package:lints/core.yaml
+    ```
 
-## Customizing the pre-defined lint sets
+## Customizing the predefined lint sets
 
 You can customize the pre-defined lint sets, both to disable one or more of the
```
Comment (pq) [inline README.md:60]: pre-defined =>
predefined

File: README.md [hunk 3]
```diff
@@ -67,7 +64,7 @@ analysis].
 ## Migrating from `package:pedantic`
 
 In migrating from the pre-defined lint sets provided by the [`package:pedantic`],
-you get static analysis coverage from 29 additional lints.  There are, however,
+you get static analysis coverage from 29 additional lints. There are, however,
 6 lints which the pedantic package includes which are not included in the
 pre-defined lint sets provided by the lints package:
```
Comment (pq) [inline README.md:69]: pre-defined =>
predefined

**Review by pq — APPROVED** [review]
👍

**Review by mit-mit — APPROVED** [review]

**Pull request merged**